test_scripts/API/VehicleData/commonVehicleData.lua

```diff
@@ -398,4 +444,45 @@ function m.processGetVDwithCustomDataSuccess()
   m.getMobileSession():ExpectResponse(cid, mobResParams)
 end
 
+function m.hmiLeveltoLimited(pAppId, pSystemContext)
+  if not pAppId then pAppId = 1 end
+  m.getHMIConnection(pAppId):SendNotification("BasicCommunication.OnAppDeactivated",
+    { appID = m.getHMIAppId(pAppId) })
+  m.getMobileSession(pAppId):ExpectNotification("OnHMIStatus",
+    { hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE", systemContext = pSystemContext })
+end
+
+function m.deactivateAppToBackground(pSystemContext)
+  if not pSystemContext then pSystemContext = "MAIN" end
+  m.getHMIConnection():SendNotification("BasicCommunication.OnEventChanged",
+    { eventName = "AUDIO_SOURCE", isActive = true })
+  m.getMobileSession():ExpectNotification("OnHMIStatus",
+    { hmiLevel = "BACKGROUND", audioStreamingState = "NOT_AUDIBLE", systemContext = pSystemContext })
+end
+
+function m.processGetVDunsuccessInvalidParams(pVehicleData, ...)
```

Comment:
@ydementieiev It is really complicate and confusing implementation. Could it be refactored to use single parameter - table like next one?
```
{{ name = "name_1", value = "value_1" }, { name = "name_2", value = "value_2" }, ...}
```

Reply:
@aderiabin check [here](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/14/commits/7c6adaabde8583e4704b7c0c7de4655059c60ef7)